### DISCUSSION TITLE
Problemas con el checkstyle

### DISCUSSION THREAD

**Discussion by sofiacortez** [dds-jv/foro#298] [Consultas de herramientas] [answered]
Hola, tengo un problema con el checkstyle, cuando hago mvn clean verify me tira BUILD ERROR y me dice que tengo 166 checkstyle violations. 

Esta es una pequeña parte de todos los que me tira (son casi todos iguales, de indentation):

[WARNING] src/main/java/calendarios/EventoUnico.java:[66] (indentation) Indentation: 'method def' child has incorrect indentation level 8, expected level should be 4.
[WARNING] src/main/java/calendarios/EventoUnico.java:[67] (indentation) Indentation: 'method def rcurly' has incorrect indentation level 4, expected level should be 2.
[WARNING] src/main/java/calendarios/EventoUnico.java:[69] (indentation) Indentation: 'method def modifier' has incorrect indentation level 4, expected level should be 2.
[WARNING] src/main/java/calendarios/EventoUnico.java:[70] (indentation) Indentation: 'method def' child has incorrect indentation level 8, expected level should be 4.
[WARNING] src/main/java/calendarios/EventoUnico.java:[71] (indentation) Indentation: 'method def' child has incorrect indentation level 8, expected level should be 4.
[WARNING] src/main/java/calendarios/EventoUnico.java:[72] (indentation) Indentation: 'method def rcurly' has incorrect indentation level 4, expected level should be 2.
[WARNING] src/main/java/calendarios/EventoUnico.java:[74] (indentation) Indentation: 'method def modifier' has incorrect indentation level 4, expected level should be 2.
[WARNING] src/main/java/calendarios/EventoUnico.java:[75] (indentation) Indentation: 'method def' child has incorrect indentation level 8, expected level should be 4.
[WARNING] src/main/java/calendarios/EventoUnico.java:[76] (indentation) Indentation: 'method def rcurly' has incorrect indentation level 4, expected level should be 2.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  5.373 s
[INFO] Finished at: 2021-07-07T15:31:58-03:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.1.2:check (Validar Formateo) on project ejercicio: You have 166 Checkstyle violations. -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException

**Comment by nanderson87** [comment, marked as answer]
Hola @sofiacortez, cómo estás?

Al parecer los errores son de identación de los archivos de código fuente. En el foro hay varias discusiones sobre el tema. Te dejo un link donde se explica un poco más.

https://github.com/dds-jv/foro/discussions/257
